static_src/components/app_container.jsx

```diff
@@ -60,12 +61,14 @@ export default class AppContainer extends React.Component {
     AppStore.addChangeListener(this._onChange);
     OrgStore.addChangeListener(this._onChange);
     SpaceStore.addChangeListener(this._onChange);
+    pageMetadataActions.loadPage('app');
```

Comment:
If we move this to `main.js`, then where would we put the unload action? That is similar to the problem I had with the breadcrumb prototype. You set a title on a page and then navigate to another page, say marketplace that doesn't set the title, you would be left with the old title.

In contrast, the container components seem like a better place to manage page lifecycle related events than the routes in `main.js` because naturally components come with their own lifecycle that aligns with the page.